src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProvider.cs

```diff
@@ -153,7 +155,8 @@ public override IReadOnlyList<RazorCompletionItem> GetCompletionItems(RazorSynta
                 var razorCompletionItem = new RazorCompletionItem(
                     completion.Key,
                     insertText,
-                    RazorCompletionItemKind.DirectiveAttribute);
+                    RazorCompletionItemKind.DirectiveAttribute,
+                    ElementCommitCharacters);
```

Comment:
Can you verify if we're still respecting pressing space or `=` to complete? If in doubt, check what happens in the old editor (with the LSP preview feature off). We'll need to be consistent with that experience.

Reply:
> still respecting pressing space or = to complete? 

No we aren't. 
- I can add `=` to `ElementCommitCharacters`, however the completion would be of the form `bind=|` (not `"|"`).
- Likewise, I can add ` ` to `ElementCommitCharacters`, but the completion is `bind |` (not `bind="|"`).

Would that be something on our end or WTE?

> If in doubt, check what happens in the old editor (with the LSP preview feature off). 

We'll need to be consistent with that experience.
For whatever reason, I'm not getting *any* directive attribute completion in this case.

Reply:
Spoke offline. We are going to try and be smart about when to add ` ` and `=` as commit characters.

Reply:
Updated to contextually support ` ` and `:` commit chars. `=` is always available. 

![MoreCommitChars](https://user-images.githubusercontent.com/14852843/80761100-c890be00-8aee-11ea-9996-71e23935ac94.gif)

Reply:
The `@attributes=|` is missing the `"|"` because we don't have snippet support enabled:
https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperCompletionService.cs#L225